src/Dropdown.js

```diff
@@ -107,11 +109,21 @@ class Dropdown extends React.Component {
     return this._$menuCtrl;
   }
 
+  getItemType() {
+    if(this.context.menuRole === 'listbox') {
+      return 'option'
+    }
+    return 'menu'
+  }
+
   getMenuItems() {
     // In a real menu with a child DropdownMenu, `this.getMenu()` should never
     // be null, but it is sometimes null in tests. To mitigate that, we just
     // use `this.getContainer()` as the fallback `menuContainer`.
     const menuContainer = this.getMenu() || this.getContainer();
+    if(this.context.menuRole) {
+      return [].slice.call(menuContainer.querySelectorAll('[role="' + this.getItemType() + '"]'));
+    }
     return [].slice.call(menuContainer.querySelectorAll('[role="menuitem"]'));
```

Comment:
```suggestion
    return [].slice.call(menuContainer.querySelectorAll(`[role="${this.getItemType()}"]`));
```

getItemType will return the default menuitem in case when menuRole doesn't exist, so we can simplify this.  Also, string interpolation is good to use here too